quilt3_local/README.md

```diff
@@ -0,0 +1,80 @@
+# Quilt3 catalog: Local development mode
+
+Open source implementation of the Quilt3 registry that works in the local
+environment (not requiring AWS cloud services).
```

Comment:
Might make sense to note that S3 Select is required, since previously I think we used endpoint from open.